src/sentry_plugins/github/plugin.py

```diff
@@ -114,6 +108,8 @@ def message_from_error(self, exc):
     def raise_error(self, exc):
         if not isinstance(exc, ApiError):
             self.logger.exception(six.text_type(exc))
+        if isinstance(exc, ApiUnauthorized):
```

Comment:
this is technically not 100% correct, they might simply not have access to the repo